website/docs/guides/1.15.0-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,64 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas Provider 1.15.0: Upgrade and Information Guide"
+sidebar_current: "docs-mongodbatlas-guides-1150-upgrade-guide"
+description: |-
+MongoDB Atlas Provider 1.15.0: Upgrade and Information Guide
+---
+
+# MongoDB Atlas Provider 1.15.0: Upgrade and Information Guide
```

Comment:
@oarbusi just FYI thanks for putting this together, few comments below. Will review again as part of v1.15.0 release PR